src/demo/CistromeExplorer.js

```diff
@@ -27,9 +27,9 @@ const demos = {
         options: {
             rowInfoAttributes: [
                 {field: "Hierarchical Clustering (Average)", type: "tree", position: "left"},
-                {field: "qc_frip", type: "quantitative", position: "left"},
-                {field: "qc_fastqc", type: "quantitative", position: "left"},
-                {field: "Metadata URL", type: "url", position: "left", title: "cid"},
+                {field: "qc_frip", type: "quantitative", position: "left", title: "QC: FRIP"},
+                {field: "qc_fastqc", type: "quantitative", position: "left", title:  "QC: FastQC"},
+                {field: "cid", type: "comparison", position: "left", title: "Compare Positive and Negative"},
```

Comment:
We could further generalize this track by having `domain` and `range` options:

```json
{..., "domain": ["positive", "negative"], "range": ["blue", "red"]}
```

In this way, colors for highlighting and texts for annotation can be defined by users. And clicking on a row multiple times will iterative through the elements defined in `domain`.

`range` can be an optional parameter, so if it is not defined by a user, we can internally use the color scheme that is being used for our `nominal` bar charts by default. 

By generalizing this, we can support other interesting functionalities, for example, allowing users to annotate samples based on their analysis results (e.g., "Worth further investigation").

Reply:
Renaming `comparison` for the clarity: Since we can consider this track as a special case of `nominal` type bar chart (we show nominal data but it can be dynamically updated by users), how about a word that can express this? `nominal-dynamic` or `user-defined-nominal`?

Reply:
I like `nominal-dynamic`